torch_tvm/register.cpp

```diff
@@ -34,7 +34,7 @@ PYBIND11_MODULE(_torch_tvm, m) {
   options.setAliasAnalysis(AliasAnalysisKind::PURE_FUNCTION);
   RegisterOperators op({Operator(
       getTVMSymbol(),
-      [](const Node* node) {
+      [](const Node* node) -> Operation {
```

Comment:
since this is breaking build, can you please split this one and land separately? It also makes review easier by not merging unrelated changes in same PR.

Reply:
Yes. I am just trying to move faster since I will out on leave soon. But if you feel strongly about it, I can split it out.